src/main/java/io/fabric8/elasticsearch/plugin/ConfigurationSettings.java

```diff
@@ -22,9 +22,6 @@
 	static final String SEARCHGUARD_AUTHENTICATION_PROXY_HEADER = "searchguard.authentication.proxy.header";
 	static final String SEARCHGUARD_CONFIG_INDEX_NAME = "searchguard.config_index_name";
 
-	static final String SEARCHGUARD_TYPE = "ac";
```

Comment:
we shouldn't need to, we rebuild the SG ac for a node whenever we start it up since the SG config is tied to the pod name